src/Ledger/Utxo/Properties.lagda

```diff
@@ -544,3 +548,198 @@ pov {tx}{utxo}{_}{fees}{deposits}{donations} h'
 \end{code}
 
 \end{property}
+
+
+\begin{property}[\textbf{General Minimum Spending Condition}]~\\
+
+\begin{code}
+
+isRefundCert : DCert → Bool
+isRefundCert (dereg c) = true
+isRefundCert (deregdrep c) = true
+isRefundCert _ = false
+
+noRefundCert : List DCert → Set _
+noRefundCert l = All (λ cert → isRefundCert cert ≡ false) l
+
+module _
+  -- ASSUMPTIONS --
+  {- 1 -} {getCoin-hom : (d₁ d₂ : DepositPurpose ⇀ Coin) → getCoin (d₁ ∪⁺ d₂) ≡ getCoin d₁ + getCoin d₂}
+  {- 2 -} {coin∅ : getCoin{A = DepositPurpose ⇀ Coin} ∅ ≡ 0}
+  {- 3 -} {⊆→∪⁺ : (d d' : DepositPurpose ⇀ Coin) → d ˢ ⊆ d' ˢ → d' ≡ d ∪⁺ d'}
+  {- 4 -} {finiteness∘listing≡id : ∀ {l} → ((proj₁ ∘ finiteness) ∘ (proj₁ ∘ listing)) l ≡ l}
```

Comment:
These new assumptions _should_ be easier to prove, but I spent quite a lot of time fighting with the set theory without success (especially assumption `{- 4 -}`) so I'm leaving them as assumptions for now, until I can resolve them myself or get help from @WhatisRT.

Reply:
I was able to prove `{- 2 -}`.

`{- 4 -}` cannot be proved as it's not necessarily true.  I replaced it with two assumptions, one for the empty list and another for a one-element list.

I have since prove it for the empty list.